packages/strapi-parts/src/DatePicker/DatePicker.stories.mdx

```diff
@@ -12,7 +13,21 @@ This is the doc of the `DatePicker` component
 
 ## DatePicker
 
-Description...
+Date pickers help users selecting a specific date from a calendar. They can be associated to a Time picker component.  
+
+**Best practices**
+
+- When the date picker is clicked, it takes today's date.
+- It is not possible to write the date from the keyboard.
+- It is not possible to change the date format. However, the date format will be based on the browser locale. 
+- A Date picker can used simultaneously with a Time picker in a form.
+- The PopOver closes after selecting a date.
+- One single format of date is currently available.
+- The Date picker is built on top on a Field component.
+
+## Usage
+
+A date picker is used to select an unique date. It's not possible to select a date range. A date picker can have a null value via the Clear feature. A navigation via the keyboard is possible within the calendar PopOver, however it's impossible to write a date from the keyboard. 
```

Comment:
I think we will need to dissociate the way we handle the data (range selection & clear data etc) and the Accessibility part (keyboard navigation). I also feel like it duplicates the best practice section because it also describes the behavior of the component.

Reply:
You are right :) We can take a lot once the Accessibility section is made, what do you think?